WordPress/src/main/res/values/strings.xml

```diff
@@ -1635,6 +1635,9 @@
     <string name="reader_bookmark_snack_title">Post saved</string>
     <string name="reader_btn_notifications_off">Turn off site notifications</string>
     <string name="reader_btn_notifications_on">Turn on site notifications</string>
+    <string name="reader_btn_remove_filter">Remove the current filter</string>
+    <string name="reader_btn_select_filter">Select a Site or Tag to filter posts</string>
+    <string name="reader_btn_selected_filter_tick">Selected</string>
```

Comment:
For these strings could you add `content_description` to the end of the name. Jayson recently created some localization guides to standardize this 😄 You can check it out [here](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/docs/localization.md)

Reply:
Changed [here](5e9a55a57258c1c032713e4781b4b5b067a94afa) and I like the document 😊, thanks for pointing to it 👍.

Reply:
Perfect!